hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -432,45 +433,55 @@
     {
       name: chip_rv_dm_cpu_debug_mem
       desc: '''Verify access to the debug mem from the CPU.
+
+            - CPU programs RW type CSRs (such as progbuf0 and progbuf1) with random value.
+            - Read back to check their values for correctness.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_rv_dm_jtag_debug_mem
       desc: '''Verify access to the debug mem from the external JTAG interface.
+
+            - Load debug ROM and read back through jtag to check the values for correctness.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_rv_dm_cpu_debug_req
       desc: '''Verify debug request to Ibex while it is actively executing.
```

Comment:
how about: "Verify that Ibex enters debug mode when triggered through JTAG"

Reply:
Updated, thanks